### ISSUE TITLE
Question about target units normalization

### ISSUE THREAD

**Issue by mailong25** [steventan0110/DiffNorm#2]
I wonder if DiffNorm is designed to normalize target speech units. Why is the  src-feat required during the training of VAE and Diffusion in the provided script? I read the paper and didn't see any mention of using source speech when training VAE and Diffusion.
.
I assume the src-feat is just a dummy argument, so I tried to set the src-feat to be the same as the tgt-feat to proceed with the training. When I perform unit generation (inference), I get the normalized units exactly the same as the original units. Any ideas?
.
Note that i got pretty high acc (~99%) on VAE task and 82% acc on Diffusion task.

**Comment by steventan0110** [comment]
Your understanding is correct. The source side is purely a dummy argument that is used by the dataloader. The dataloader will load source data that will never be used during VAE and Diffusion Training. Sorry for this hacky code!

High accuracy is expected for VAE and Diffusion tasks (though diffusion accuracy varies based on your start time) and your result looks reasonable.
